pkgs/os-specific/linux/xpadneo/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, fetchFromGitHub, kernel, bluez }:
+
+stdenv.mkDerivation rec {
+  pname = "xpadneo";
+  version = "0.8";
+
+  src = fetchFromGitHub {
+    owner = "atar-axis";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0dlcl771wc3zwnw70pxskx0yr9safjgiwps533z20gclw6v5zdpp";
+  };
+
+  setSourceRoot = ''
+    export sourceRoot=$(pwd)/source/hid-xpadneo/src
+  '';
+
+  nativeBuildInputs = kernel.moduleBuildDependencies;
+  buildInputs = [ bluez ];
+
+  makeFlags = [
+    "-C"
+    "${kernel.dev}/lib/modules/${kernel.modDirVersion}/build"
+    "M=$(sourceRoot)"
+  ];
+
+  buildFlags = [ "modules" ];
+  installFlags = [ "INSTALL_MOD_PATH=$(out)" ];
```

Comment:
```suggestion
  installFlags = [ "INSTALL_MOD_PATH=${placeholder "out"}" ];
```

Reply:
:+1: Will do. I used `$(out)` because I saw that most other modules use it, though I see why it's better to use placeholder.